src/models/album.rs

```diff
@@ -26,6 +26,16 @@ impl Album {
         album::table.filter(album::id.eq(id)).first::<Self>(conn)
     }
 
+    pub fn get_artwork_url(id: &str) -> String {
+        format!("/files/artwork/{}/raw", id)
+    }
+
+    pub fn artwork_url(&self) -> Option<String> {
+        self.artwork_path
+            .as_ref()
+            .map(|_| Album::get_artwork_url(&self.id.to_string()))
```

Comment:
This can be simplified into:
```rust
if self.artwork_path.is_some() {
    Some(Album::get_artwork_url(&self.id.to_string()))
} else {
    None
}
```